Features/Stories/Fieldtypes/AuthorsFieldType.feature

```diff
@@ -0,0 +1,175 @@
+Feature: Test the validations done on fields from PlatformUI - Authors fieldtype
+    In order to validate the authors fieldtype
+    As an Editor  user
+    I need to be able to create, update and delete content with authors fieldtypes
+
+    Background:
+        Given I am logged in as an Editor in PlatformUI
+
+    ##
+    # Validate the existence of expected fields from a field type when creating a content
+    ##
+    @javascript
+    Scenario: A Content of a Content Type that has an author fieldtype must have an author and an email fields
+        Given a Content Type with an "authors" Field exists
+        When I create a content of this type
+        Then I should see a "name" field
+        And I should see an "email" field
+
+    @javascript
+    Scenario: A Content of a Content Type that has an author fieldtype must be prefilled with user's name and email
+        Given a Content Type with an "authors" Field exists
+        When I create a content of this type
+        Then I should see a "name" field filled with Editor's Username
+        And I should see an "email" field filled with Editor's Email
+
+    @javascript
+    Scenario: The label of an mandatory author field of a Content must be marked as mandatory
+        Given a Content Type with a "required" "authors" with "Name" "Participants" exists
+        When I create a content of this type
+        Then the "Participants" field should be marked as mandatory
+
+    @javascript
+    Scenario: A Content of a Content Type that has an author fieldtype must have a "Remove this author" and a "Add another author" buttons
+        Given a Content Type with an "authors" Field exists
+        When I create a content of this type
+        Then I should see a "Remove this author" button
+        And I should see an "Add another author" button
+
+    @javascript
+    Scenario: When editing a Content the label of an author field must have the same name than field type from the respective Content Type
+        Given a Content Type with an "authors" with "Name" "Participants" exists
+        When I create a content of this type
+        Then I should see a "Participants:" label related with the "name" and "email" fields

```

Comment:
I'm not a fan of the usage *related* here and in the lines below because the *relation* between "Participants" and the "name" and "email" fields is different from the *relation* between "Name:" and the "name" field and *related* is not precise enough.

Actually, I would replace *Then I should see a "Participants:" label related with the "name" and "email" fields* by something like *Then I should see "Participants" as the field definition name*

And for the other case, I would replace *And I should see a "Name:" label related with the "name" field* by *And I should see a form field labelled with "Name:"*
and replace *And I should see an "Email:" label related with the "email" field* by *And I should see a form field labelled with "Email:"*